src/styles/type-scale/index.md

```diff
@@ -0,0 +1,476 @@
+            text: "Used by"
+          },
+          {
+            text: "Font size",
+            format: "numeric"
+          },
+          {
+            text: "Line height",
+            format: "numeric"
+          }
+        ],
+        rows: [
+          [
+            {
+              text: "80"
+            },
+            {
+              text: "Only used in exceptional circumstances"
+            },
+            {
+              text: "53px",
+              format: "numeric"
+            },
+            {
+              text: "55px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "48"
+            },
+            {
+              html: "<code>govuk-heading-xl<code>"
+            },
+            {
+              text: "32px",
+              format: "numeric"
+            },
+            {
+              text: "35px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "36"
+            },
+            {
+              html: "<code>govuk-heading-l<code>"
+            },
+            {
+              text: "27px",
+              format: "numeric"
+            },
+            {
+              text: "30px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "27"
+            },
+            {
+              text: "Only used in exceptional circumstances"
+            },
+            {
+              text: "21px",
+              format: "numeric"
+            },
+            {
+              text: "25px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "24"
+            },
+            {
+              html: "<code>govuk-heading-m</code>, <code>govuk-body-l</code>"
+            },
+            {
+              text: "21px",
+              format: "numeric"
+            },
+            {
+              text: "25px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "19"
+            },
+            {
+              html: "<code>govuk-heading-s</code>, <code>govuk-body</code>"
+            },
+            {
+              text: "19px",
+              format: "numeric"
+            },
+            {
+              text: "25px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "16"
+            },
+            {
+              html: "<code>govuk-body-s<code>"
+            },
+            {
+              text: "16px",
+              format: "numeric"
+            },
+            {
+              text: "20px",
+              format: "numeric"
+            }
+          ]
+        ]
+      }) }}
+
+      The scale for large screens has not changed.
+  </div>
+</details>
+
+## Using the type scale in your own CSS
+
+Include the `govuk-font` mixin to use GOV.UK typography in your CSS. 
```

Comment:
Could we link to the [API docs](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-font) here?

Reply:
Decision: do it!